mod_form.php

```diff
@@ -338,6 +338,10 @@ private function bigbluebuttonbn_mform_add_block_room_room(&$mform, $cfg) {
         if ($cfg['recording_editable']) {
             $field['type'] = 'checkbox';
             $field['description_key'] = 'mod_form_field_record';
+            // Try to check the opencast config and make sure that opencast is available for this course.
+            if ($cfg['oc_recording'] && bigbluebuttonbn_check_opencast(get_course($this->current->course)->id)) {
+                $field['description_key'] = 'mod_form_field_record_oc';
```

Comment:
Personally, I don't think that the teacher who is editing the activity needs to know that the recording is processed in Opencast. I think you could leave the standard string even if the `oc_recording` option is enabled.